arm-compute/quickstart-templates/aliases.json

```
@@ -16,34 +16,69 @@
             "version": "latest",
             "architecture": "x64"
           },
+          "CentOS85Gen2":  {
```

only one space between: and {, please modify the following spaces like this
```suggestion
          "CentOS85Gen2": {
```